go/vt/sqlparser/sql.y

```diff
@@ -1486,13 +1496,13 @@ drop_statement:
     // Change this to an alter statement
     $$ = &DDL{Action: AlterStr, Table: $5}
   }
-| DROP VIEW exists_opt table_name ddl_skip_to_end
+| DROP VIEW exists_opt view_name_list
   {
     var exists bool
         if $3 != 0 {
           exists = true
         }
-    $$ = &DDL{Action: DropStr, FromTables: TableNames{$4.ToViewName()}, IfExists: exists}
+    $$ = &DDL{Action: DropStr, FromTables: $4, IfExists: exists}
```

Comment:
Should be FromViews?

Reply:
Thanks! The fact that this now works is also reflected in tests :).